src/three-render-objects.js

```diff
@@ -361,10 +365,28 @@ export default Kapsule({
       state.objects.forEach(obj => state.scene.add(obj)); // Add to scene
     }
 
-    //
-
     function finishLoad() {
       state.loadComplete = state.scene.visible = true;
     }
   }
-});
\ No newline at end of file
+});
+
+const THREE_JS_PERSPECTIVE_CAMERA_FOV_Y_DEFAULT = 50;
+
+function updateCamera(camera, width, height, depth) {
```

Comment:
Small nit, `depth` could be extracted from `camera.position.z` eliminating the need for the additional argument.